src/c++/perf_analyzer/genai-perf/tests/test_cli.py

```diff
@@ -247,6 +247,35 @@ def test_default_profile_export_filepath(
         captured = capsys.readouterr()
         assert captured.out == ""
 
+    @pytest.mark.parametrize(
+        "arg, expected_path",
+        [
+            (
+                ["--model", "strange/test_model"],
+                "artifacts/strange_test_model-triton-tensorrtllm-concurrency1",
+            ),
+            (
+                [
+                    "--model",
+                    "hello/world/test_model",
+                    "--service-kind",
+                    "openai",
+                    "--endpoint-type",
+                    "chat",
+                ],
+                "artifacts/hello_world_test_model-openai-chat-concurrency1",
+            ),
+        ],
+    )
+    def test_model_name_artifact_path(self, monkeypatch, arg, expected_path, capsys):
+        combined_args = ["genai-perf"] + arg
+        monkeypatch.setattr("sys.argv", combined_args)
+        args, extra_args = parser.parse_args()
+
+        assert args.artifact_dir == Path(expected_path)
+        captured = capsys.readouterr()
+        assert captured.out == ""
```

Comment:
Shouldn't the log be captured? (Ex: "Model name '{args.model}' cannot be used to create artifact directory. Instead, '{filtered_name}' will be used.")

Reply:
@nv-hwoo found the cause of this. We use the debug level for the logger in these tests. We could probably have a different ticket that moves the logger level to info. We could use the status code and/or search for error messages to check whether an error happened.